src/components/Picklist-item.vue

```diff
@@ -47,7 +47,7 @@ export default defineComponent({
 </script>
 <style>
 
-ion-note{
```

Comment:
why can't we just select the element? why do we have to use a class? @sanskar345

Reply:
We can do it I just previously not used scoped style so added a CSS class. Will do it.

Reply:
@dt2patel  Changes done